cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -429,9 +429,25 @@ runTxBuildCmd
               let consensusMode = consensusModeOnly cModeParams
               case consensusMode of
                 CardanoMode -> do
-                  (nodeEraUTxO, _, eraHistory, systemStart, _) <-
-                    queryStateForBalancedTx_ nodeEra nid allTxInputs
-                      & handleQueryConvenienceErrors_
+                  (nodeEraUTxO, eraHistory, systemStart) <-
+                    (do
+                      let cModeParams2 = CardanoModeParams $ EpochSlots 21600
```

Comment:
Is there any reason to not use `cModeParams`?

Reply:
No it should be fine. Did it not compile when you tried to use the `localNodeConnInfo` from the higher scope?

Reply:
It compiles.  Updated to use the `localNodeConnInfo` from the the higher scope.